support-frontend/assets/pages/paper-subscription-landing/components/content/linkTo.tsx

```diff
@@ -34,14 +37,16 @@ function LinkTo({
 }: {
 	setTabAction: (arg0: PaperFulfilmentOptions) => void;
 	tab: PaperFulfilmentOptions;
-	children: Node;
+	children: ReactNode;
 	activeTab?: PaperFulfilmentOptions | null;
 	isPricesTabLink?: boolean;
-}) {
+}): JSX.Element {
+	console.log('tab --->', tab);
+
 	return (
 		<Link
 			css={isPricesTabLink ? linkStyles(tab, activeTab) : linkColor}
-			href={paperSubsUrl(tab === 'delivery')}
+			href={paperSubsUrl(tab === 'HomeDelivery')}
```

Comment:
Interestingly I thought I'd picked up a bug here. The code was incorrect as `tab` is the type `PaperFulfilmentOptions` which can only ever hold the value "HomeDelivery" or "Collection". I fixed the code and the went looking for the bug, but I found it never manifests itself to the user...

`linkTo` is used for the bottom tabs on the Newspaper landing page, the bottom tabs are anchor tags but we don't use the `href` property of these anchors to route a user to a new page when they click on them, instead we have an `onClick` handler that take control, the actions of the handler are correct - the correct Home Delivery or Collection content is displayed in the page. This differs from the way the top tabs on the Newspaper landing page behave, these are also anchor tags but they route a user through to either `/subscribe/paper` or `subscribe/paper/delivery`.

**Top tabs**
<img width="607" alt="Screenshot 2022-04-01 at 14 48 50" src="https://user-images.githubusercontent.com/1590704/161278551-aaee9cf0-6567-49b6-9e04-7b320882e89f.png">


**Bottom tabs**
<img width="406" alt="Screenshot 2022-04-01 at 14 49 04" src="https://user-images.githubusercontent.com/1590704/161278466-a370a476-9147-4d69-b6e0-a1665c140046.png">

Reply:
Weird 🤔 I feel like it's not great if the two separate sets of tabs have slightly different behaviour, but as long as the user sees the correct thing I suppose it's not a big deal.

Reply:
I know, I don't think we should have 2 sets of behaviours either. I'm going to create a card to look into consolidating one way or the other.